specs/eventing/data-plane.md

```diff
@@ -1,125 +1,184 @@
-event sender MAY ignore an event reply in the delivery response if the
-`Prefer: reply` header was not included in the delivery request.
-
-An example is that a Broker supporting event reply sends events with an
-additional header `Prefer: reply` so that the sink connected to the Broker knows
-event replies will be accepted. While a source sends events without the header,
-in which case the sink may assume that any event reply will be dropped without
-error or retry attempt. If a sink wishes to ensure the reply events will be
-delivered, it can check for the existence of the `Prefer: reply` header in the
-delivery request and respond with an error code if the header is not present.
-
-### Data plane contract for Sources
-
-See [Source Delivery specification](sources.md#source-event-delivery)
-for details.
-
-### Data plane contract for Channels
-
-See [Channel Delivery specification](channel.md#data-plane) for details.
-
-### Data plane contract for Brokers
-
-See [Broker Delivery specification](broker.md)
-
-## Changelog
-
-- 2020-04-20: `0.13.x release`: initial version that documents common contract
-  for sinks, sources, channels and brokers.
+Event senders MAY add or update CloudEvents attributes before sending to
+implement observability features such as tracing; in particular, the
+`traceparent` and `tracestate` distributed tracing attributes defined by
+[W3C](https://www.w3.org/TR/trace-context/) and
+[CloudEvents](https://github.com/cloudevents/spec/blob/v1.0/extensions/distributed-tracing.md)
+MAY be modified in this way for each delivery attempt of the same event.
+
+This specification does not mandate any particular logging or metrics
+aggregation, nor a method of exposing observability information to users
+configuring the resources. Platform administrators SHOULD expose event-delivery
+telemetry to users through platform-specific interfaces, but such interfaces are
+beyond the scope of this document.
+
+### Derived (Reply) Events
+
+In some applications, an event recipient MAY emit an event in reaction to a
+received event. Senders MAY choose to support this pattern by accepting an
+encoded CloudEvent in the HTTP response.
+
+An event sender MAY document support for this pattern by including a
+`Prefer: reply` header in the HTTP POST request. This header indicates to the
+event recipient that the caller will accept a
+[`200` response](#event-acknowledgement-and-repeat-delivery) which includes a
+CloudEvent encoded using the binary or structured formats.
+[Brokers](./overview.md#broker) and [Channels](./overview.md#channel) MUST
+indicate support for replies using the `Prefer: reply` header when sending to
+the `spec.subscriber` address.
+
+A recipient MAY reply to any HTTP POST with a `200` response to indicate that
+the event was processed successfully, with or without a response payload. If the
+recipient will _never_ provide a response payload, the `202` response code is
```

Comment:
the "\_never\_ " is a bit odd. I don't think we should based things on whether or not the sink will EVER respond with something other than a 202, rather focus just on what the sender should do in response to THIS message flow. So, in this case just saying: "If there's no reply then a 202 MAY be returned"
should be sufficient.

Reply:
Done.